src/server/routes/entity/edition.js

```diff
@@ -151,6 +160,14 @@ router.get(
 				addInitialRelationship(props, relationshipTypeId, initialRelationshipIndex, props.publication);
 			}
 
+			if (props.work) {
+				initialState.editionSection.work = props.work;
```

Comment:
The .work property does not exist in editionSection, contrarily to how .publication and .publisher are set.
Have a look at https://github.com/bookbrainz/bookbrainz-site/blob/c4a48cb0709456407dfbb098d9ca32f6b4ac8344/src/server/routes/entity/edition.js#L235-L251

Reply:
This `initialState.editionSection.work = props.work;` does not serve any purpose, as there is nowhere a .work property is used in the editionSection.